<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Katana (0.1a) Application Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/settings-icon.png</normaloff>:/icons/settings-icon.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="generalSettings">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QCheckBox" name="enableAnimations">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enable User Interface Animations</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSlider" name="animationsLevel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>0</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="enableAutoUpdates">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>23</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enable Automatic Updates</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="autoUpdateAction">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>23</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>*Select Automatic Update Action</string>
        </property>
        <item>
         <property name="text">
          <string>*Select Update Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Download and Install</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Download and Ask</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ask</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ignore</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="mediaSettings">
       <attribute name="title">
        <string>Media</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="networkSettings">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Using network Proxy:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="proxyBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>441</width>
          <height>141</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Proxy Server Configuration</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Proxy Server Address:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="proxyAddress">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="proxyUser">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>80</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>30</y>
           <width>41</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="proxyPort">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>30</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>80</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="proxyPass">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>80</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QCheckBox" name="useAnnonConnection">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>53</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Connect Annonymously</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="socksV4">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>110</y>
           <width>116</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Socks V4</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="socksV5">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>110</y>
           <width>116</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Socks V5</string>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="useProxy">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Use Proxy Networking</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
